libs/ui/src/diagnostics/configuration_section.tsx

```diff
@@ -93,7 +132,10 @@ export function ConfigurationSection({
       {!(expectPrecinctSelection && !precinctSelection) && (
         <P>
           <SuccessIcon /> Ballot Styles:{' '}
-          {getBallotStyleIds(election, precinctSelection).join(', ')}
+          {getBallotStylesConfigurationDetails(
+            election,
+            precinctSelection
+          ).join(', ')}
```

Comment:
This might be a bit easier to read as a list or a table, e.g.:

Ballot Styles:
- 1 - English, Chinese, Spanish (US)
- 2 - English, Chinese, Spanish (US)

 Ballot Styles
| ID | Language |
| --- | --------- |
| 1  | English, Chinese, Spanish (US) |
| 2  | English, Chinese, Spanish (US) |

Reply:
Yeah I thought about this but wasn't sure with printing the report if it would be too long, will mock up and get Matt's perspective though